lua/packer/util.lua

```diff
@@ -110,14 +110,14 @@ util.float = function()
   local left = math.ceil((columns - width) * 0.5)
   local top = math.ceil((lines - height) * 0.5 - 1)
 
-  local opts = {
+  opts = vim.tbl_deep_extend('keep', {
```

Comment:
@gbrlsnchs I think you might want to use `force` here since presumably this should prefer the user's options over packer's?

Reply:
> "keep": use value from the leftmost map
from the help docs, this means the left side map takes priority, so you won't actually be able to override anything explicitly set in packer

Reply:
Hi @akinsho, thanks for the review. My initial idea was to not be able to override what Packer explicitly sets so one wouldn't be able to mess with the size of the window. Do you think it would be better if a user could do such?

Reply:
I guess it means that a user can completely mess with the window but ultimately it's their prerogative 🤷🏾‍♂️ I think if we give them the ability to change this we should just give them full control since it only affects their system and who knows what someone might prefer.

They might want to put the window in the bottom right of the editor and completely mess with the height. More importantly I think it will be confusing to people that they can pass in some settings and those work but others just don't.

Reply:
I think that makes total sense. I'll rebase with the proper parameter. Thanks for the insight.